apps/ui/src/views/Proposal/Votes.vue

```diff
@@ -209,26 +210,28 @@ watch([sortBy, choiceFilter], () => {
               </div>
             </td>
             <td class="relative">
-              <div
-                v-if="proposal.type !== 'basic'"
-                class="truncate"
-                :title="proposal.choices[vote.choice - 1]"
-              >
-                {{ proposal.choices[vote.choice - 1] }}
-              </div>
-              <UiButton
-                v-else
-                class="!w-[40px] !h-[40px] !px-0 cursor-default bg-transparent"
-                :class="{
-                  '!text-skin-success !border-skin-success': vote.choice === 1,
-                  '!text-skin-danger !border-skin-danger': vote.choice === 2,
-                  '!text-gray-500 !border-gray-500': vote.choice === 3
-                }"
-              >
-                <IH-check v-if="vote.choice === 1" class="inline-block" />
-                <IH-x v-else-if="vote.choice === 2" class="inline-block" />
-                <IH-minus-sm v-else class="inline-block" />
-              </UiButton>
+              <Encrypted :encrypted="isEncrypted">
```

Comment:
What I don't really understand is why you need to wrap this inside `Encrypted` can't you simply use `v-if=isEncrypted` and `v-else`?